### ISSUE TITLE
error: PyramidFlowSampler Error while processing rearrange-reduction pattern "b c t h w -> (b t) c h w". Input tensor shape: torch.Size([16, 1, 96, 160]). Additional info: {}. Wrong shape: expected 5 dims. Received 4-dim tensor.

### ISSUE THREAD

**Issue by lijackcoder** [kijai/ComfyUI-PyramidFlowWrapper#19]
not sure what this is

PyramidFlowSampler
Error while processing rearrange-reduction pattern "b c t h w -> (b t) c h w".
Input tensor shape: torch.Size([16, 1, 96, 160]). Additional info: {}.
Wrong shape: expected 5 dims. Received 4-dim tensor.

 ComfyUI Error Report
## Error Details
- **Node Type:** PyramidFlowSampler
- **Exception Type:** einops.EinopsError
- **Exception Message:**  Error while processing rearrange-reduction pattern "b c t h w -> (b t) c h w".
 Input tensor shape: torch.Size([16, 1, 96, 160]). Additional info: {}.
 Wrong shape: expected 5 dims. Received 4-dim tensor.
## Stack Trace
```
  File "C:\AI\pinokio\api\comfy.git\app\execution.py", line 323, in execute
    output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)

  File "C:\AI\pinokio\api\comfy.git\app\execution.py", line 198, in get_output_data
    return_values = _map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)

  File "C:\AI\pinokio\api\comfy.git\app\execution.py", line 169, in _map_node_over_list
    process_inputs(input_dict, i)

  File "C:\AI\pinokio\api\comfy.git\app\execution.py", line 158, in process_inputs
    results.append(getattr(obj, func)(**inputs))

  File "C:\AI\pinokio\api\comfy.git\app\custom_nodes\ComfyUI-PyramidFlowWrapper\nodes.py", line 213, in sample
    latents = model["model"].generate_i2v(

  File "C:\AI\pinokio\api\comfy.git\app\env\lib\site-packages\torch\utils\_contextlib.py", line 116, in decorate_context
    return func(*args, **kwargs)

  File "C:\AI\pinokio\api\comfy.git\app\custom_nodes\ComfyUI-PyramidFlowWrapper\pyramid_dit\pyramid_dit_for_video_gen_pipeline.py", line 408, in generate_i2v
    clean_latents_list = self.get_pyramid_latent(torch.cat(generated_latents_list, dim=2), len(stages) - 1)

  File "C:\AI\pinokio\api\comfy.git\app\env\lib\site-packages\torch\utils\_contextlib.py", line 116, in decorate_context
    return func(*args, **kwargs)

  File "C:\AI\pinokio\api\comfy.git\app\custom_nodes\ComfyUI-PyramidFlowWrapper\pyramid_dit\pyramid_dit_for_video_gen_pipeline.py", line 169, in get_pyramid_latent
    x = rearrange(x, 'b c t h w -> (b t) c h w')

  File "C:\AI\pinokio\api\comfy.git\app\env\lib\site-packages\einops\einops.py", line 591, in rearrange
    return reduce(tensor, pattern, reduction="rearrange", **axes_lengths)

  File "C:\AI\pinokio\api\comfy.git\app\env\lib\site-packages\einops\einops.py", line 533, in reduce
    raise EinopsError(message + "\n {}".format(e))

**Comment by lijackcoder** [comment]
I think it works now after i git pulled

**Comment by kijai** [comment]
This was my mistake when I was updating stuff, fixed it already as you noticed.
